src/app/portlets/dot-edit-content/services/dot-container-contentlet.service.ts

```diff
@@ -1,18 +1,24 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
+import { CoreWebService } from 'dotcms-js/core/core-web.service';
+import { RequestMethod } from '@angular/http';
 
 @Injectable()
 export class DotContainerContentletService {
-    constructor() {}
+    constructor(private coreWebService: CoreWebService) {}
 
     getContentletToContainer(containerId: string, contentletId: string): Observable<string> {
```

Comment:
I have a this method too in the edit, so probably will be conflict in the merge.